apps/condo/domains/meter/schema/RegisterMetersReadingsService.test.js

```diff
@@ -889,29 +889,14 @@ describe('RegisterMetersReadingsService', () => {
 
     })
 
-    test('Meter info controlReadingsDate should take todays date on empty input if meter is being created', async () => {
```

Comment:
Should not take

Reply:
It’s better to rewrite test for new case

Reply:
Yes, as other dates. See the next test.
<img width="633" alt="image" src="https://github.com/user-attachments/assets/c938a885-2238-4265-b63e-90c6246fef9a">